server/ApiGateway/Properties/launchSettings.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:10188",
+      "sslPort": 44375
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
```

Comment:
Just out of curiosity, is this configuration used when launching the app on Docker? If so, isn't this attempt to run browser carried out?

Reply:
I am not really sure how it works, but I don't think it's used when launching on Docker (at least it didn't crash when I deleted entire file :P ). Anyway, I think that launching browser is not really necessary here